x/utilities/bimap_with_default.go

```diff
@@ -0,0 +1,19 @@
+package utilities
+
+import "github.com/vishalkuo/bimap"
+
+func GetWithDefault(bimap *bimap.BiMap, k interface{}, defaultValue interface{}) interface{} {
```

Comment:
GetWithDefault() name should make sense in context of the package name.

Reply:
Moved the package.